README.md

```diff
@@ -217,6 +217,8 @@ yarn run dev
 
 If changes aren't detected, you might need to increase `fs.inotify.max_user_watches`. See <https://webpack.js.org/configuration/watch/#not-enough-watchers>. If you need to increase your watchers, it's common to see multiple errors beginning with `Error from chokidar`.
 
+Note:  ensure `yarn run dev` has finished its initial build before visiting http://localhost:9000, otherwise `./bin/bridge` will stop running.
```

Comment:
```suggestion
> [!NOTE]
> Ensure `yarn run dev` has finished its initial build before visiting http://localhost:9000, otherwise `./bin/bridge` will stop running.
```

wdyt of using the alerts in GitHub flavored markdown?

Reply:
Good suggestion.  I am inclined to leave it as is given we're not currently using those, and I it will add the length of the already long document.

Reply:
ok, fair point

/lgtm